created stellar_deployment_guide.md file

mimiprosper opened the pull request:
Create a README.md for Contracts in StarShop in the StarShop:

In this Documentation I made a step-by-step instructions and simplified setup process of deploying Smart Contracts using Stellar for new developers

The content of this documentation covers  guides:

Prerequisites: Necessary tools and dependencies.

Environment Setup:

Configuring Stellar Smart Contracts.
- Specific steps to install related tools or plugins.

Wallet Configuration:
- Instructions to connect and configure the wallet.

Compilation and Deployment:
- Detailed explanation for compiling and deploying Smart Contracts.

Testing and Execution:
- A practical, verified example of a deployed Smart Contract.

Troubleshooting:
Tips and solutions for common errors during the setup process.

aguilar1x reviewed (commented):
I need you to make everything more visually friendly, so that reading doesn't become a burden. Both at the level of code in markdown, and dividing the texts in a better way, titles and subtitles, categories, numbering, etc. As a recommendation, a good practice to make things look better visually is to add emojis.

I also need you to summarize everything more, since it has a lot of information, very complete but very extensive. Therefore, summarize it more and get to the point.

mimiprosper commented:
I have made some update on the Docs:

- More visual appealing with some emoji
- Added summaries
- Restructured the contents
- simplified some guideline

PLEASE NOTE:
This Doc is extension. There are limit to how this Docs can be summaried so it doesnt lose content.
Markdown files are not HTML files. There are limitations on how Markdown files can be structured.

aguilar1x reviewed (approved):
LGTM! 🚀

Thanks for contributing to the project!
